src/components/NewWorkspaceModal.js

```diff
@@ -99,18 +99,25 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
       onSuccess(await Utils.cond(
         [cloneWorkspace, async () => {
           const workspace = await Ajax().Workspaces.workspace(cloneWorkspace.workspace.namespace, cloneWorkspace.workspace.name).clone(body)
+          if (isAzureBillingProject() && !getConfig().isProd) {
```

Comment:
doesn't `isAzureBillingProject` require a project passed in as an argument?

Reply:
No it does not (see implementation above, which gets the current project if a project is not passed in).